### PR TITLE
Injet PORT for LB startup

### PR DESCRIPTION
**Pull request by siddhipai**
@superkhau PTAL

### CODE REVIEW

File: test/model.application-credential.test.js [hunk 1]
```diff
@@ -33,12 +33,12 @@ describe('ApplicationCredential', function() {
       done(err);
     });
   });
-
+var port = process.env.PORT || 0;
```
Comment (superkhau) [inline test/model.application-credential.test.js]: || 3000;

**Comment by superkhau** [comment]
And rebase please

**Comment by Amir-61** [comment]
@slnode test please

**Comment by superkhau** [comment]
@Amir-61 Go ahead, it hasn't been started yet.

File: test/model.application-credential.test.js (hunk 1 above)
Comment (superkhau) [inline test/model.application-credential.test.js]: Actually move this down to line 38.

**Comment by superkhau** [comment]
@Amir-61 Can you submit a new PR and ping me for review? Let's close this PR since its been left on the backburner for awhile.

**Pull request closed without merging**